src/app/test/service-stubs.ts

```diff
@@ -261,19 +263,26 @@ export class HttpStubService {
 }
 
 export class WorkflowStubService {
-    nsWorkflows$ = observableOf([]);
+    nsWorkflows$ = Observable.of([]);
+    nsSharedWorkflows$ = Observable.of([]);
     workflow$: BehaviorSubject<any> = new BehaviorSubject({}); // This is the selected workflow
     workflowId$ = observableOf(1);
     workflows$: BehaviorSubject<Workflow[]> = new BehaviorSubject([]);  // This contains the list of unsorted workflows
-    copyBtn$ = observableOf({});
+    sharedWorkflows$: BehaviorSubject<Workflow[]> = new BehaviorSubject([]);  // This contains the list of unsorted workflows
+    copyBtn$ = Observable.of({});
```

Comment:
all of these when rxjs compat is removed:  error TS2339: Property 'of' does not exist on type 'typeof Observable'.